src/runtime/components/Divider.vue

```diff
@@ -0,0 +1,69 @@
+<script lang="ts">
+import { tv, type VariantProps } from 'tailwind-variants'
+import type { SeparatorProps } from 'radix-vue'
+import type { AvatarProps } from './Avatar.vue'
+import type { AppConfig } from '@nuxt/schema'
+import _appConfig from '#build/app.config'
+import theme from '#build/ui/divider'
+
+const appConfig = _appConfig as AppConfig & { ui: { divider: Partial<typeof theme> } }
+
+const divider = tv({ extend: tv(theme), ...(appConfig.ui?.divider || {}) })
+
+type DividerVariants = VariantProps<typeof divider>
+
+export interface DividerProps extends Omit<SeparatorProps, 'asChild'> {
+  avatar?: AvatarProps
+  class?: any
+  color?: DividerVariants['color']
+  decorative?: boolean
+  icon?: string
+  label?: string
+  orientation?: DividerVariants['orientation']
+  size?: DividerVariants['size']
+  type?: DividerVariants['type']
+  ui?: Partial<typeof divider.slots>
```

Comment:
```suggestion
  label?: string
  icon?: string
  avatar?: AvatarProps
  decorative?: boolean
  orientation?: DividerVariants['orientation']
  color?: DividerVariants['color']
  size?: DividerVariants['size']
  type?: DividerVariants['type']
  class?: any
  ui?: Partial<typeof divider.slots>
```

Reply:
any tips on how I can follow this order on other components too?